### DISCUSSION TITLE
Methods for OAuth2 authentication

### DISCUSSION THREAD

**Discussion by Tom-Vroemen** [Azure/data-api-builder#1898] [Q&A]
Hi,

We are currently setting up an API on top of our Datawarehouse for a client of ours, While setting up OAuth 2 we see that the only documented method is using AZ CLI, this works fine however are there any alternatives to creating an Entra ID token?

**Comment by seantleonard** [comment]
Hi @Tom-Vroemen , You can configure DAB to use an Azure AD identify provider. Guidance can be found here: https://learn.microsoft.com/en-us/azure/data-api-builder/authentication-azure-ad
